test/controllers/authorisedOfficials/AuthorisedOfficialsPassportControllerSpec.scala

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.authorisedOfficials
+
+import java.time.LocalDate
+
+import base.SpecBase
+import controllers.actions.{AuthIdentifierAction, FakeAuthIdentifierAction}
+import forms.common.DateOfBirthFormProvider
+import models.{Index, Name, NormalMode, Passport, SelectTitle, UserAnswers}
+import navigation.AuthorisedOfficialsNavigator
+import navigation.FakeNavigators.FakeAuthorisedOfficialsNavigator
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.{reset, verify, _}
+import org.scalatest.BeforeAndAfterEach
+import pages.authorisedOfficials.{AuthorisedOfficialsNamePage, AuthorisedOfficialsPassportPage}
+import play.api.data.Form
+import play.api.inject.bind
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.test.Helpers._
+import repositories.UserAnswerRepository
+import views.html.common.{DateOfBirthView, PassportView}
+
+import scala.concurrent.Future
+
+class AuthorisedOfficialsPassportControllerSpec extends SpecBase with BeforeAndAfterEach {
+
+  override lazy val userAnswers: Option[UserAnswers] = Some(emptyUserAnswers)
+
+  override def applicationBuilder(): GuiceApplicationBuilder =
+    new GuiceApplicationBuilder()
+      .overrides(
+        bind[UserAnswerRepository].toInstance(mockUserAnswerRepository),
+        bind[AuthorisedOfficialsNavigator].toInstance(FakeAuthorisedOfficialsNavigator),
+        bind[AuthIdentifierAction].to[FakeAuthIdentifierAction]
+      )
+
+  override def beforeEach(): Unit = {
+    super.beforeEach()
+    reset(mockUserAnswerRepository)
+  }
+
+  private val messageKeyPrefix = "authorisedOfficialsPassport"
+  private val view: PassportView = inject[PassportView]
+  private val formProvider: DateOfBirthFormProvider = inject[DateOfBirthFormProvider]
+  private val form: Form[LocalDate] = formProvider(messageKeyPrefix)
+
+  private val controller: AuthorisedOfficialsPassportController = inject[AuthorisedOfficialsPassportController]
+
+  private val requestArgs = Seq("passportNumber" -> "123", "country" -> "gb", "expiryDate.year" -> "2021", "expiryDate.month" -> "1", "expiryDate.day" -> "1")
```

Comment:
this will start failing in 3 months - might be a good idea to make the year something like LocalDate.now().getYear + 1?